README.md

```diff
@@ -47,48 +51,71 @@ end
 Available configuration options include:
 
 * `access_key`: The AWS access key ID that has access to SNS publishing and/or
-  SQS subscribing.  *(required)*
```

Comment:
This is going to be a [hard habit to break](http://www.quickanddirtytips.com/education/grammar/two-spaces-after-a-period).  :cry: